.travis.yml

```diff
@@ -0,0 +1,19 @@
+dist: xenial
```

Comment:
What about kind of periodic workflow?

Reply:
As far as I understand it is determined by BUILD_OS variable passed do docker container. Then build will we executed for darwin as well.